up & going/ch2.md

```diff
@@ -400,20 +400,20 @@ foo();
 
 Note que `c` não está disponível dentro de `bar()`, porque está declarado dentro do escopo de `baz()`, e o `b` não está disponível para `foo()` pelo mesmo motivo.
 
-Se você tentar acessar o valor da variável dentro de um escopo onde ela não está disponível, você irá receber um erro de `ReferenceError`. Se você tentar setar uma variável que ainda não foi declarada, ou você terminará criando uma variável no escopo global (ruim!) ou gerar um erro, dependendo de você ter declarado "strict mode" (veja "Strict Mode"). Vamos dar uma olhada:
+Se você tentar acessar o valor da variável dentro de um escopo onde ela não está disponível, você irá receber um erro de `ReferenceError`. Se você tentar setar uma variável que ainda não foi declarada, ou você terminará criando uma variável no escopo global (ruim!) ou irá gerar um erro. Para isso você precisa ter declarado "strict mode" (veja "Strict Mode"). Vamos dar uma olhada:
 
 ```js
 function foo() {
-    a = 1;  // `a` not formally declared
+    a = 1;  // `a` não foi formalmente declarada
 }
 
 foo();
-a;          // 1 -- oops, auto global variable :(
+a;          // 1 -- oops, você acaba de criar uma variável global automática :(
 ```
 
-Esta é uma prática muito ruim. Não faa isso! Sempre declare suas variáveis formalmente.
+Esta é uma prática muito ruim. Não faça isso! Sempre declare suas variáveis formalmente.
 
-Além de criarmos declarações de variáveis no mesmo nível da função, o ES6 *deixa* (let) você criar variáveis que irão pertencer os blocos individuais (pares de `{ .. }`), usando a palavra-chave `let`. Apesar de suas nuances e detalhes, as regras do escopo terão o comportamento bem parecido com o que vimos em funções.
+Além de criarmos declarações de variáveis no mesmo nível da função, o ES6 possuie o (let) que deixa você criar variáveis que irão pertencer a blocos individuais (pares de `{ .. }`), usando a palavra-chave `let`. Apesar de suas nuances e detalhes, as regras do escopo terão o comportamento bem parecido com o que vimos em funções.
```

Comment:
@gmsecrieru o trocadilho ao qual você se refere é em relação a tradução de _let_? Se for isso também concordo em manter a versão anterior.